fuel-core/src/coin_query.rs

```diff
@@ -1,549 +1,672 @@
 use crate::{
```

Comment:
Should we rename `coin_query` to `resource_query`?

Reply:
Yes, forgot about it=)